src/CMakeLists.txt

```diff
@@ -51,6 +133,12 @@ if (APPLE AND NOT GGML_NO_ACCELERATE)
     endif()
 endif()
 
+# use OpenBLAS
+if (OPENBLAS)
+    set(CMAKE_C_FLAGS  "${CMAKE_C_FLAGS} -DGGML_USE_OPENBLAS -I/usr/local/include/openblas")
+    set(CMAKE_LD_FLAGS  "${CMAKE_LD_FLAGS} -lopenblas")
+endif()
+
```

Comment:
Is this needed?

We already have this:

https://github.com/ggerganov/whisper.cpp/blob/b4ebdb6b57dedf9e82eec6e13ac70e8089e06190/CMakeLists.txt#L102-L114

Reply:
No, I'm sorry for my less-confirmation.